codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/ServerServiceGenerator.kt

```diff
@@ -36,6 +36,7 @@ class ServerServiceGenerator(
      */
     fun render() {
         val operations = index.getContainedOperations(context.serviceShape).sortedBy { it.id }
+        // TODO for loop
```

Comment:
Out of curiosity what is wrong with this map?

Reply:
A pet peeve of mine: `.map` should be used to map a value to another value _without side-effects_. As written this returns `Unit` and has lots of side effects. A for loop communicates intent better.

Reply:
Got it.